modules/ROOT/pages/bonita-version-update-index.adoc

```diff
@@ -1,25 +1,40 @@
-= Bonita version update 
-Learn how to upgrade a Bonita Platform from an older version to a newer version.
+= Bonita update and upgrade
+:description: Learn how to update a Bonita Studio or Runtime from an older version to a newer version, or transition from our Community to our Enterprise edition.
+
+{description}
 
 [.card-section]
-== Upgrading your platform to the new version
+== Updating to a new Bonita version
+
+You can automatically update Bonita from an earlier version, provided it is not older than 6.0.2. +
+In the case of Bonita Runtime, after the update, a process continues to run unchanged. Later on, you can modify the process definition to take advantage of the new features in the new version of Bonita.
```

Comment:
in the new version of Bonita. => in the new Bonita version. 

formulation proposal.

Reply:
replaced. you can use the suggestion feature (+ and - in a sheet), very handy.